crates/router/src/db/business_profile.rs

```diff
@@ -444,7 +447,7 @@ impl ProfileInterface for MockDb {
             .transpose()?
             .ok_or(
                 errors::StorageError::ValueNotFound(format!(
-                    "No business profile found for profile_name = {profile_name} and merchant_id = {merchant_id:?}"
+                    "No business profile found for profile_name = {profile_name:?} and merchant_id = {merchant_id:?}"
```

Comment:
Can we check how the `Debug` impl looks like, and ensure we don't include something like `NameType("profile_name")` in user-facing error messages? We should ideally only include the underlying string values in user-facing error messages.